### DISCUSSION TITLE
Anyone have any luck with the If-Modified-Since header?

### DISCUSSION THREAD

**Discussion by Reboot4X** [weather-gov/api#709] [General]
I've added this to my API request (https://api.weather.gov//gridpoints/MTR/81,137), but always get back a 200 (ok).   
I'm using the date programmatically based on the last update time of a previous api call:   
e.g. 
  If-Modified-Since: Mon, 04 Mar 2024 20:26:53 GMT

In this case, I'm pretty sure it should be returning the 304 (Not Modified) code back, but it is always reporting a 200. 

Here's part of the response for context

    "properties": {
        "@id": "https://api.weather.gov/gridpoints/MTR/81,137",
        "@type": "wx:Gridpoint",
        "updateTime": "2024-03-04T20:26:52+00:00",
        "validTimes": "2024-03-04T14:00:00+00:00/P7DT14H",

Anyone run into similar issues?  Any other dependencies with using the if-modified-since header?  (I found one old issue from 2021, where someone ran into the same issue, but it was never resolved.)

Thanks,

-Shawn

**Comment by sullynole** [comment]
The Last-Modified header currently is not in use on the gridpoints endpoints, and the if-modified-since requests are being ignored.

This info can be found on the Updates tab here: https://www.weather.gov/documentation/services-web-api

> 25 Feb 2022: The gridpoints endpoint no longer serves stale data. For this endpoint we currently are not returning the Last-Modified header. In addition, the If-Modified-Since header is also being ignored for gridpoints request.
